raisinbread/RB_files/RB_Config.sql

```diff
@@ -92,5 +92,6 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (95,49,'1000');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (96,100,'environment');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (97,70,'/data-raisinbread/');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (98,93,'V1');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (99,101,'%MINCToolsPath%');
```

Comment:
```suggestion
INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (99,101,'');
```

Maybe this can just be blank? I think this format is only used as a convention for the `sed` command in the `install.sh` script and similar places.

Reply:
sure, I'll change it to empty string then. Can always change it back if needed.

Reply:
yeah, it was just to give sed something to search for.